### PR TITLE
Feature/retrieve sfids from polygon

### PR DESCRIPTION
**Pull request by AdrienJaussaudGeosys**

### CODE REVIEW

File: geosyspy/geosys.py
Comment (jpn-geo6) [inline geosyspy/geosys.py:1]: - missing attributes description in docstring
For example: geometry (str) : the input geometry 
- missing type in signature (geometry, season_fields_ids )

File: geosyspy/services/master_data_management_service.py
Comment (jpn-geo6) [inline geosyspy/services/master_data_management_service.py:1]: missing doctstring

File: tests/test_int_geosys.py
Comment (jpn-geo6) [inline tests/test_int_geosys.py:1]: assert on the len of the result can be dangerous: the test could fail if  a new sfd is created in the geometry.
len = 20 seems to fit with the default limit of the response api ( which has been removed with the $limit=none)

**Review by jpn-geo6 — COMMENTED** [review]
missing few docstring , type in attributes & a test could fail in the future

**Comment by AdrienJaussaudGeosys** [comment]
updated PR

**Review by jpn-geo6 — APPROVED** [review]

**Pull request merged**